DQMOffline/Configuration/python/DQMOffline_SecondStep_cff.py

```diff
@@ -85,19 +85,26 @@
                                            DQMMessageLoggerClientSeq )
 
 
-DQMHarvestCommon = cms.Sequence( dqmRefHistoRootFileGetter *
-                                 DQMMessageLoggerClientSeq *
-                                 dqmDcsInfoClient *
-                                 SiStripOfflineDQMClient *
-                                 TrackingOfflineDQMClient *
-                                 PixelOfflineDQMClientNoDataCertification *
-                                 triggerOfflineDQMClient *
-                                 hltOfflineDQMClient *
-                                 dqmFEDIntegrityClient *
-                                 alcaBeamMonitorClient *
-                                 runTauEff *
-                                 dqmFastTimerServiceClient
-                                )
+from DQMOffline.L1Trigger.L1TStage2CaloLayer2Efficiency_cfi import l1tStage2CaloLayer2Efficiency
+from DQMOffline.L1Trigger.L1TStage2CaloLayer2Diff_cfi import l1tStage2CaloLayer2EmuDiff
+# l1tStage2CaloLayer2EmuDiff uses plots produced by l1tStage2CaloLayer2Efficiency
+DQMHarvestL1Trigger = cms.Sequence(l1tStage2CaloLayer2Efficiency * l1tStage2CaloLayer2EmuDiff)
```

Comment:
its likely better to move this sequence definition  into DQMOffline/L1Trigger - its easy to see that it will expand in the future